### PR TITLE
Add docs linking to the online builder

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Added docs linking to the online builder. Part of #1174.

---

### Additional information

I added the link to the online builder in the `Installation` section. However, I'm not sure if it's the best place to add it. Or if there are better places.

@wwalc suggested the [Custom builds guide](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/development/custom-builds.html), but to me, the online builder doesn't fit the *development* section of this guide and the guide itself focusing mostly on forking GH's `ckeditor5-build-*` repos.

cc @Reinmar @wwalc

### CODE REVIEW

**Comment by wwalc** [comment]
Custom builds have been for many months/years the only option that we had for people looking for a way to download a customized build. For example even [https://ckeditor.com/ckeditor-5/download/](https://ckeditor.com/ckeditor-5/download/) links there.  
  
Because of this, we have lots of external links pointing to this place (think e.g. an external tutorial about CKEditor 5, Stackoverflow and so on). Which means that we should use this page too, to promote the online builder. Something like:  
  
_If you are looking for an easy way to create a custom build of CKEditor 5, check also the_ [_online builder_](https://ckeditor.com/ckeditor-5/online-builder/)_, which allows you to create easily a custom build through a simple and intuitive UI._

The Installation article is of course also a good place. The thing is that we may/should link to the online builder in as many places as needed/

**Comment by wwalc** [comment]
[https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/installing-plugins.html](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/installing-plugins.html) is another place to advertise the online builder. See for example the docs for CKEditor 4:

*   [https://ckeditor.com/docs/ckeditor4/latest/features/autogrow.html](https://ckeditor.com/docs/ckeditor4/latest/features/autogrow.html) 
*   [https://ckeditor.com/docs/ckeditor4/latest/guide/dev\_widget\_installation.html#online-builder-installation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_widget_installation.html#online-builder-installation)
*   [https://ckeditor.com/docs/ckeditor4/latest/guide/dev\_plugins.html#online-builder-installation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_plugins.html#online-builder-installation)
*   [https://ckeditor.com/docs/ckeditor4/latest/examples/basicpreset.html](https://ckeditor.com/docs/ckeditor4/latest/examples/basicpreset.html)

(there are tons of links everywhere) to get a feeling on how much we advertise the online builder for CKEditor 4 (the first box). It's beneficial for both: us and users if they are pointed to the simpler tool first whenever there is a high chance they will start asking a question okay, how to add plugins.

**Comment by ma2ciek** [comment]
I've added links in these two places. I've read other sections and I didn't find more places to add links. We could create a separate docs for the online builder itself, but TBH it's pretty simple and straightforward. Could you check the new version?

File: docs/builds/guides/integration/installation.md [hunk 1]
```diff
@@ -44,6 +45,10 @@ npm install --save @ckeditor/ckeditor5-build-decoupled-document
 
 CKEditor will then be available at `node_modules/@ckeditor/ckeditor5-build-[name]/build/ckeditor.js`. It can also be imported directly to your code by `require( '@ckeditor/ckeditor5-build-[name]' )`.
 
+### Online builder
+
+Go to the [Online builder page](https://ckeditor.com/ckeditor-5/online-builder/), go through a few creator steps to pick plugins and toolbar items and visualize the editor. Then download the prepared build with a simple instruction how to run it.
```
Comment (wwalc) [inline docs/builds/guides/integration/installation.md]: The [online builder](https://ckeditor.com/ckeditor-5/online-builder/) lets you download CKEditor 5 builds and also allows you to create your own, customized builds (with a different set of plugins) in a few easy steps, through a simple and intuitive UI.

Comment (ma2ciek) [inline docs/builds/guides/integration/installation.md]:
```suggestion
The [online builder](https://ckeditor.com/ckeditor-5/online-builder/) lets you download CKEditor 5 builds and also allows you to create your own, customized builds (with a different set of plugins) in a few easy steps, through a simple and intuitive UI.
```

**Comment by ma2ciek** [comment]
> I'd modify just one thing in the installation instruction, the rest is okay.

Done.

**Comment by wwalc** [comment]
Looks good 👍

**Comment by ma2ciek** [comment]
@Reinmar, could you merge it?

File: docs/builds/guides/development/custom-builds.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Some of the reasons for creating custom builds are:
 * Changing the {@link features/ui-language localization language} of the editor.
 * Enabling bug fixes which are still not a part of any public release.
 
+If you are looking for an easy way to create a custom build of CKEditor 5, check also the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), which allows you to create easily a custom build through a simple and intuitive UI.
```
Comment (dkonopka) [inline docs/builds/guides/development/custom-builds.md]: Shouldn't we distinguish this information with `<info-box hint>`?

Comment (ma2ciek) [reply docs/builds/guides/development/custom-builds.md]: Maybe it's a good idea.

File: docs/builds/guides/integration/installing-plugins.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ In this guide you can learn how to add plugins to your editor in the two most co
 * When you use an {@link builds/guides/overview editor build},
 * When you {@link framework/guides/quick-start build your editor from source}.
 
+<info-box hint>
+If you are looking for an easy way to create a custom build of CKEditor 5 without installing anything, check the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), which allows you to create easily a build with a custom set of plugins through a simple and intuitive UI.
```
Comment (Reinmar) [inline docs/builds/guides/integration/installing-plugins.md]:
```suggestion
	If you are looking for an easy way to create a custom build of CKEditor 5 without installing anything, check the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), which allows you to create easily a build with a custom set of plugins through a simple and intuitive UI.
```

File: docs/builds/guides/development/custom-builds.md [hunk 4]
```diff
@@ -21,6 +21,10 @@ Some of the reasons for creating custom builds are:
 * Changing the {@link features/ui-language localization language} of the editor.
 * Enabling bug fixes which are still not a part of any public release.
 
+<info-box hint>
+If you are looking for an easy way to create a custom build of CKEditor 5, check also the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), which allows you to create easily a custom build through a simple and intuitive UI.
```
Comment (Reinmar) [inline docs/builds/guides/development/custom-builds.md]:
```suggestion
	If you are looking for an easy way to create a custom build of CKEditor 5, check also the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), which allows you to create easily a custom build through a simple and intuitive UI.
```

**Pull request merged**